src/views/EditSimple.vue

```diff
@@ -22,12 +22,12 @@
 
 <template>
 	<Popover ref="popover"
-		:open="isVisible"
+		:shown="isVisible"
 		:auto-hide="false"
 		:placement="placement"
-		:boundaries-element="boundaryElement"
-		open-class="event-popover"
-		trigger="manual">
+		:boundary-element="boundaryElement"
```

Comment:
This is only called boundary
```suggestion
		:boundary="boundaryElement"
```